src/algebra/lie/subalgebra.lean

```diff
@@ -137,12 +138,23 @@ instance : lie_ring_module L' M :=
 
 @[simp] lemma coe_bracket_of_module (x : L') (m : M) : ⁅x, m⁆ = ⁅(x : L), m⁆ := rfl
 
+variables [module R M] [lie_module R L M]
+
 /-- Given a Lie algebra `L` containing a Lie subalgebra `L' ⊆ L`, together with a Lie module `M` of
 `L`, we may regard `M` as a Lie module of `L'` by restriction. -/
-instance [module R M] [lie_module R L M] : lie_module R L' M :=
+instance : lie_module R L' M :=
 { smul_lie := λ t x m, by simp only [coe_bracket_of_module, smul_lie, submodule.coe_smul_of_tower],
   lie_smul := λ t x m, by simp only [coe_bracket_of_module, lie_smul], }
 
+/-- An `L`-equivariant map of Lie modules `M → N` is `L'`-equivariant for any Lie subalgebra
+`L' ⊆ L`. -/
+def restrict_lie_module_hom (f : M →ₗ⁅R,L⁆ N) : M →ₗ⁅R,L'⁆ N :=
+{ map_lie' := λ x m, f.map_lie ↑x m,
+  .. (f : M →ₗ[R] N)}
```

Comment:
We already have `linear_map.restrict_scalars` which means something similar - so maybe this should be called `lie_module_hom.restrict_lie` or `lie_module_hom.restrict_lie_ring` both to enable dot notation and for consistency

Reply:
Agreed. I've just added https://github.com/leanprover-community/mathlib/pull/7591/commits/b132960c36f4e2e016d92a3011423a885b56b920 to cater for this.